demisto_sdk/commands/common/errors.py

```diff
@@ -4350,3 +4355,10 @@ def duplicated_script_name(
             "is replaced by the word Incident/Incidents\nfor example: if there is a script `getIncident'"
             "it will not be possible to create a script with the name `getAlert`)"
         )
+
+    @staticmethod
+    @error_code_decorator
+    def hidden_pack_not_mandatory_dependency(
+        hidden_pack: str, dependant_packs_ids: Set[str]
+    ):
+        return f"Cannot hide pack {hidden_pack} because it is a mandatory dependency for packs {', '.join(dependant_packs_ids)}"
```

Comment:
```suggestion
        return f"Cannot hide pack {hidden_pack} because it is a mandatory dependency for the following packs: {', '.join(dependant_packs_ids)}"
```